### ISSUE TITLE
pupil_player crashes, pupil_capture works

### ISSUE THREAD

**Issue by dschreij** [pupil-labs/pupil#208]
I have a strange issue with the latest version of pupil_player 0.6.11 on our Ubuntu 15.04 laptop, which we use for research with the eyetracker. pupil_capture starts without problems, but pupil_player crashes with the following traceback:

```python
Traceback (most recent call last):
  File "<string>", line 92, in <module>
  File "/usr/local/lib/python2.7/dist-packages/PyInstaller-2.1.1dev_67610f2_mod-py2.7.egg/PyInstaller/loader/pyi_importers.py", line 276, in load_module
  File "/home/mkassner/pupil/deploy_player/build/bundle/out00-PYZ.pyz/video_capture", line 23, in <module>
  File "/usr/local/lib/python2.7/dist-packages/PyInstaller-2.1.1dev_67610f2_mod-py2.7.egg/PyInstaller/loader/pyi_importers.py", line 420, in load_module
ImportError: libavcodec.so.54: cannot open shared object file: No such file or directory
```
I had a look in /opt/pupil_player folder and it appears to contain libavcodec.so.56 (which is also the  version present in the Ubuntu 15.04 repositories). This same version is also present in /opt/pupil_capture, but capture doesn't appear to trip over this. What would your recommendation be? Try do downgrade to libavcodec.so.54 (even though this is not included it the distribution repositories anymore)? Thanks for any help with this.

**Comment by mkassner** [comment]
I have updated Pupil Player and Capture about 60mins ago with a new version that should address the issues you have been having. Could you give Pupil Player version 06.113 a try?

https://github.com/pupil-labs/pupil/releases/download/v0.6.10/pupil_player_0.6-113_Linux_x64.deb

**Comment by mkassner** [comment]
Hi,  did you get a chance to test this?

**Comment by mkassner** [comment]
Please reopen or make new issue if this is still a problem!

Thanks!

Moritz

**Comment by dschreij** [comment]
Sorry I was on holidays! I will test it on Monday when I get back to work.

**Comment by dschreij** [comment]
I tested it and this has solved the problem! Pupil player works again (0.6-170)
Thanks!
